scripts/renovate-bump-version.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
```

Comment:
shellcheck has opinions on the script:

```bash
[Line 16:](javascript:setPosition(16, 28))
version=$(grep '^version:' ${parentDir}/Chart.yaml | awk '{print $2}')
                           ^-- [SC2086](https://www.shellcheck.net/wiki/SC2086) (info): Double quote to prevent globbing and word splitting.

Did you mean: ([apply this](javascript:applyFixIndex([0])), apply [all SC2086](javascript:applyFixCode(2086)))
version=$(grep '^version:' "${parentDir}"/Chart.yaml | awk '{print $2}')
 
[Line 17:](javascript:setPosition(17, 14))
major=$(echo $version | cut -d. -f1)
             ^-- [SC2086](https://www.shellcheck.net/wiki/SC2086) (info): Double quote to prevent globbing and word splitting.

Did you mean: ([apply this](javascript:applyFixIndex([1])), apply [all SC2086](javascript:applyFixCode(2086)))
major=$(echo "$version" | cut -d. -f1)
 
[Line 18:](javascript:setPosition(18, 14))
minor=$(echo $version | cut -d. -f2)
             ^-- [SC2086](https://www.shellcheck.net/wiki/SC2086) (info): Double quote to prevent globbing and word splitting.

Did you mean: ([apply this](javascript:applyFixIndex([2])), apply [all SC2086](javascript:applyFixCode(2086)))
minor=$(echo "$version" | cut -d. -f2)
 
[Line 19:](javascript:setPosition(19, 14))
patch=$(echo $version | cut -d. -f3)
             ^-- [SC2086](https://www.shellcheck.net/wiki/SC2086) (info): Double quote to prevent globbing and word splitting.

Did you mean: ([apply this](javascript:applyFixIndex([3])), apply [all SC2086](javascript:applyFixCode(2086)))
patch=$(echo "$version" | cut -d. -f3)
 
[Line 20:](javascript:setPosition(20, 9))
patch=$(expr $patch + 1)
        ^-- [SC2003](https://www.shellcheck.net/wiki/SC2003) (style): expr is antiquated. Consider rewriting this using $((..)), ${} or [[ ]].
             ^-- [SC2086](https://www.shellcheck.net/wiki/SC2086) (info): Double quote to prevent globbing and word splitting.

Did you mean: ([apply this](javascript:applyFixIndex([5])), apply [all SC2086](javascript:applyFixCode(2086)))
patch=$(expr "$patch" + 1)
 
[Line 21:](javascript:setPosition(21, 62))
sed -i "s/^version:.*/version: ${major}.${minor}.${patch}/g" ${parentDir}/Chart.yaml
>>                                                           ^-- [SC2086](https://www.shellcheck.net/wiki/SC2086) (info): Double quote to prevent globbing and word splitting.

Did you mean: ([apply this](javascript:applyFixIndex([6])), apply [all SC2086](javascript:applyFixCode(2086)))
sed -i "s/^version:.*/version: ${major}.${minor}.${patch}/g" "${parentDir}"/Chart.yaml
 
[Line 24:](javascript:setPosition(24, 34))
appVersion=$(grep '^appVersion:' ${parentDir}/Chart.yaml | awk '{print $2}')
                                 ^-- [SC2086](https://www.shellcheck.net/wiki/SC2086) (info): Double quote to prevent globbing and word splitting.

Did you mean: ([apply this](javascript:applyFixIndex([7])), apply [all SC2086](javascript:applyFixCode(2086)))
appVersion=$(grep '^appVersion:' "${parentDir}"/Chart.yaml | awk '{print $2}')
 
[Line 25:](javascript:setPosition(25, 48))
sed -i -e '/^  artifacthub.io\/changes: |/,$d' ${parentDir}/Chart.yaml
                                               ^-- [SC2086](https://www.shellcheck.net/wiki/SC2086) (info): Double quote to prevent globbing and word splitting.

Did you mean: ([apply this](javascript:applyFixIndex([8])), apply [all SC2086](javascript:applyFixCode(2086)))
sed -i -e '/^  artifacthub.io\/changes: |/,$d' "${parentDir}"/Chart.yaml
 
[Line 26:](javascript:setPosition(26, 1))
echo "  artifacthub.io/changes: |" >> ${parentDir}/Chart.yaml
^-- [SC2129](https://www.shellcheck.net/wiki/SC2129) (style): Consider using { cmd1; cmd2; } >> file instead of individual redirects.
                                      ^-- [SC2086](https://www.shellcheck.net/wiki/SC2086) (info): Double quote to prevent globbing and word splitting.

Did you mean: ([apply this](javascript:applyFixIndex([10])), apply [all SC2086](javascript:applyFixCode(2086)))
echo "  artifacthub.io/changes: |" >> "${parentDir}"/Chart.yaml
 
[Line 27:](javascript:setPosition(27, 31))
echo "    - kind: changed" >> ${parentDir}/Chart.yaml
                              ^-- [SC2086](https://www.shellcheck.net/wiki/SC2086) (info): Double quote to prevent globbing and word splitting.

Did you mean: ([apply this](javascript:applyFixIndex([11])), apply [all SC2086](javascript:applyFixCode(2086)))
echo "    - kind: changed" >> "${parentDir}"/Chart.yaml
 
[Line 28:](javascript:setPosition(28, 65))
echo "      description: Bump ${chartName} to ${appVersion}" >> ${parentDir}/Chart.yaml
>>                                                              ^-- [SC2086](https://www.shellcheck.net/wiki/SC2086) (info): Double quote to prevent globbing and word splitting.

Did you mean: ([apply this](javascript:applyFixIndex([12])), apply [all SC2086](javascript:applyFixCode(2086)))
echo "      description: Bump ${chartName} to ${appVersion}" >> "${parentDir}"/Chart.yaml
 
[Line 29:](javascript:setPosition(29, 5))
cat ${parentDir}/Chart.yaml
    ^-- [SC2086](https://www.shellcheck.net/wiki/SC2086) (info): Double quote to prevent globbing and word splitting.

Did you mean: ([apply this](javascript:applyFixIndex([13])), apply [all SC2086](javascript:applyFixCode(2086)))
cat "${parentDir}"/Chart.yaml
```

Reply:
I will have a look into it. (Probably tomorrow)

Reply:
It's also part of megalinter if that's a thing you're into. 

We use it the argo training material repo if you want to see how. (See the comment it nicely adds to the PR): https://github.com/argoproj-labs/training-material/pull/46

Reply:
> It's also part of megalinter if that's a thing you're into.
> 
> We use it the argo training material repo if you want to see how. (See the comment it nicely adds to the PR): [argoproj-labs/training-material#46](https://github.com/argoproj-labs/training-material/pull/46)

This thing looks awesome :D

Reply:
It's nice. The only downside is it's amd64 only, which annoys us internally as we are an arm shop wherever possible.

Reply:
Oh and unsurprisingly the docker images are pretty huge if you self host.